app/router.cjsx

```diff
@@ -13,7 +13,7 @@ React = require 'react'
 `import ProjectStatus from './pages/admin/project-status';`
 `import EditMediaPage from './pages/lab/media';`
 `import UserProfilePage from './pages/profile/index';`
-`import NotificationsPage from './pages/notifications';`
+`import NotificationsPage from './pages/notifications/index';`
```

Comment:
You don't need to specify `index` here. An import of a folder will load any file named index first. (This generally works for web file structure and isn't just for node). Not sure why others did that above in the imports.